sources/de-bkg.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "Service Centre of the Federal Government for Geo-Information and Geodesy",
+    "edtf:inception": "2019-05-10",
+    "fullname": "The Service Centre of the Federal Government for Geo-Information and Geodesy (DLZ) and the Central Office for Geo-Topography (Zentrale Stelle Geotopographie, ZSGT) of the Laender",
+    "id": 1377649075,
+    "key": "",
+    "license": "https://www.govdata.de/dl-de/by-2-0",
+    "license_text": "The data and meta-data provided may, for commercial and non-commercial use, in particular be copied, printed, presented, altered, processed and transmitted to third parties; be merged with own data and with the data of others and be combined to form new and independent datasets; be integrated in internal and external business processes, products and applications in public and non-public electronic networks.",
+    "license_type": "Data licence Germany – attribution – Version 2.0",
+    "name": "de-bkg",
+    "prefix": "de-bkg",
```

Comment:
I'm fine with this prefix of `de-bkg`, but let's say what it is (`Bundesamt für Kartographie und Geodäsie`) somewhere in the source, preferably the `fullname`, and move that huge string into a different property like the description?

Reply:
Taken care of in: https://github.com/whosonfirst/whosonfirst-sources/pull/148/commits/2bf028478089159f471961d6171878e79ec11b2c